tests/after_cluster_deploy_sanity/test_after_cluster_deploy_sanity.py

```
@@ -85,13 +88,15 @@ def test_hyperconverged_sanity(admin_client, hco_namespace):
 
 @pytest.mark.cluster_health_check
 def test_storage_sanity(cluster_storage_classes_names):
+    """Test that all required storage classes from configuration exist on the cluster."""
     assert storage_sanity_check(cluster_storage_classes_names=cluster_storage_classes_names)
 
 
 @pytest.mark.cluster_health_check
 def test_boot_volume_health(
     golden_images_namespace, hco_managed_data_import_crons, data_import_cron_managed_datasources
```

Ruff (0.14.14)

[warning] Unused function argument: `golden_images_namespace`

(ARG001)